### ISSUE TITLE
[Bug]: Unexpected character '0' is added to broken text by util.breakText function.

### ISSUE THREAD

**Issue by DavidSama1993** [clientIO/joint#2252]
## Input  
![code screenshot](https://github.com/clientIO/joint/assets/30044890/11127d0f-cdfa-4ef8-8354-55e2117f08b7)

## Output
![resultScreenshot](https://github.com/clientIO/joint/assets/30044890/9be8d8fb-5019-4358-98a5-53e0b46cc9c1)

I have tried to debug the problem and my solution is bellow.
![solution](https://github.com/clientIO/joint/assets/30044890/c484fcdf-88c0-470e-9f68-8e3d2138fbd0)

### Version

3.6.2

### What browsers are you seeing the problem on?

Chrome

### What operating system are you seeing the problem on?

Windows

**Comment by kumilingus** [comment]
- Does the issue exist in the latest version `3.7`?
- Could you please paste here the input text?
- Why is there a `text` attribute on the `<text>` element in your screenshot? JointJS does not add the `text` attribute when rendering text.

**Comment by DavidSama1993** [comment]
+ I didn't try it in the lastest version, but I have viewed the source code in v3.7.4 and found that there is no difference at the position I marked. So I guess the problem remains.
+ The input text could be `'aaaaaaaaaaaaaa/ddddd\n金木水火土'` and the `'\n'` is a linefeed.
+ I'm so sorry that I don't know why there is a `text` attribute on the `<text>` element. I just set the `attrs` according to the method demonstrated in the official document like this. The unexpected charactor '0' is occasional when I resize the element.
![image](https://github.com/clientIO/joint/assets/30044890/b989369a-0a63-4015-b46e-b859c9f179ec)
